### PR TITLE
feat: Changed Youtube and Github color Code and add my data

### PR DESCRIPTION
**Pull request by RoshanMhatre**
Add feature Youtube and Github Color and Add My Data

## Proposed Changes
Changed Youtube color from red to #ff0000 and Github from black to #171515
Add my linkfree data
  -
  -

## Screenshots

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/150"></a>

### CODE REVIEW

File: public/data/roshanmhatre.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        "icon": "github"
+      },
+      {
+        "name": "My Twitter",
+        "url": "https://twitter.com/RoshanMhatre",
+        "icon": "twitter"
+      }
+    ]
+  }
+
```
Comment (eddiejaoude) [inline public/data/roshanmhatre.json:18]: GitHub recommends an empty blank line at the end of each file, displayed with the red icon

File: src/Components/Links.js [hunk 2]
```diff
@@ -5,9 +5,9 @@ import { Button } from 'primereact/button'
 
 function Links({ links }) {
   const colors = {
-    youtube: 'red',
+    youtube: '#FF0000',
     twitter: '#00ACEE',
-    github: 'black',
+    github: '#171515',
```
Comment (eddiejaoude) [inline src/Components/Links.js:10]: Thank you 👍 btw next time this would be better as a separate PR - plus twice the green squares 🤓

**Review by eddiejaoude — APPROVED** [review]
Looks good 👍

File: public/data/roshanmhatre.json (hunk 1 above)
Comment (RoshanMhatre) [reply public/data/roshanmhatre.json:18]: Ok Great! Thanks for letting me know, will keep in mind next time.
